src/main/java/com/bettercloud/vault/api/Logical.java

```diff
@@ -162,4 +150,29 @@ public LogicalResponse write(final String path, final Map<String, String> nameVa
         }
     }
 
+    private LogicalResponse buildResponseFromContent(RestResponse restResponse, int retryCount) throws VaultException {
```

Comment:
hi - it is used below (line 177):
``
return new LogicalResponse(restResponse, retryCount, data);
``